packages/api/cms-api/src/auth/dto/current-user.ts

```diff
@@ -1,4 +1,13 @@
-export class CurrentUser {
+export interface CurrentUser {
     id: string;
     role: string | undefined;
+    contentScopes?: Array<Record<string, string>>;
 }
+
+/*
```

Comment:
Do you intend to keep this comment?

Reply:
yes, it might make it a bit clearer what this looks like